python/ray/worker.py

```diff
@@ -292,7 +292,7 @@ def store_and_register(self, object_id, value, depth=100):
                                        .format(type(e.example_object)))
                     print(warning_message)
                 except serialization.RayNotDictionarySerializable:
-                    _register_class(type(e.example_object), pickle=True)
+                    _register_class(type(e.example_object), pickled=True)
```

Comment:
Is there a reason you renamed pickle to pickled? As it is it is slightly inconsistent...

Reply:
Because cloudpickle is imported as pickle, which causes a conflict in `_register_class`. I could make a separate function for register class if this is an issue.